<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointCloudLabClass</class>
 <widget class="QMainWindow" name="PointCloudLabClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PointCloudLab</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_drag">
         <property name="text">
          <string>Drag</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pointPick">
         <property name="text">
          <string>PointPick</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_areaPick">
         <property name="text">
          <string>AreaPick</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_undo">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_allSelect">
         <property name="text">
          <string>AllSelect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_invertSelect">
         <property name="text">
          <string>InvertSelect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_setting">
         <property name="text">
          <string>Setting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>TestButt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newBtn">
         <property name="text">
          <string>newBtn</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>885</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QVTKWidget" name="qvtkWidget">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>打开的点云</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTreeWidget" name="treeWidget">
               <property name="mouseTracking">
                <bool>true</bool>
               </property>
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <column>
                <property name="text">
                 <string notr="true">1</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>输出</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTextBrowser" name="textBrowser">
               <property name="acceptRichText">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openFileAction"/>
    <addaction name="saveFileAction"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="filterAction1"/>
    <addaction name="filterAction2"/>
    <addaction name="filterAction3"/>
    <addaction name="filterAction4"/>
   </widget>
   <widget class="QMenu" name="menuFit">
    <property name="title">
     <string>Fit</string>
    </property>
    <addaction name="linefitAction"/>
    <addaction name="planefitAction"/>
    <addaction name="ballfitAction"/>
   </widget>
   <widget class="QMenu" name="menuMatch">
    <property name="title">
     <string>Match</string>
    </property>
    <addaction name="match1Action"/>
    <addaction name="match2Action"/>
    <addaction name="match3Action"/>
   </widget>
   <widget class="QMenu" name="menuMesh">
    <property name="title">
     <string>Mesh</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMeasure">
    <property name="title">
     <string>Measure</string>
    </property>
    <addaction name="pmdAction"/>
    <addaction name="boundaryAction"/>
   </widget>
   <widget class="QMenu" name="menuRoi">
    <property name="title">
     <string>ROI</string>
    </property>
    <addaction name="copyPointAction"/>
    <addaction name="extractPointAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilter"/>
   <addaction name="menuFit"/>
   <addaction name="menuMatch"/>
   <addaction name="menuMesh"/>
   <addaction name="menuMeasure"/>
   <addaction name="menuRoi"/>
  </widget>
  <action name="openFileAction">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="filterAction1">
   <property name="text">
    <string>直通滤波</string>
   </property>
  </action>
  <action name="filterAction2">
   <property name="text">
    <string>体素滤波
</string>
   </property>
  </action>
  <action name="filterAction3">
   <property name="text">
    <string>统计滤波</string>
   </property>
  </action>
  <action name="copyPointAction">
   <property name="text">
    <string>复制所选点云</string>
   </property>
  </action>
  <action name="extractPointAction">
   <property name="text">
    <string>提取所选点云</string>
   </property>
  </action>
  <action name="actionjkj">
   <property name="text">
    <string>jkj</string>
   </property>
  </action>
  <action name="match1Action">
   <property name="text">
    <string>点云粗配准</string>
   </property>
  </action>
  <action name="filterAction4">
   <property name="text">
    <string>投影滤波</string>
   </property>
  </action>
  <action name="linefitAction">
   <property name="text">
    <string>直线拟合</string>
   </property>
  </action>
  <action name="planefitAction">
   <property name="text">
    <string>平面拟合</string>
   </property>
  </action>
  <action name="boundaryAction1">
   <property name="text">
    <string>边缘提取</string>
   </property>
  </action>
  <action name="pmdAction1">
   <property name="text">
    <string>平面度计算</string>
   </property>
  </action>
  <action name="match2Action">
   <property name="text">
    <string>点云精配准</string>
   </property>
  </action>
  <action name="ballfitAction">
   <property name="text">
    <string>三维球拟合</string>
   </property>
  </action>
  <action name="pmdAction">
   <property name="text">
    <string>平面度计算</string>
   </property>
  </action>
  <action name="boundaryAction">
   <property name="text">
    <string>边缘提取</string>
   </property>
  </action>
  <action name="match3Action">
   <property name="text">
    <string>点云拼接</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PointCloudLab.qrc"/>
 </resources>
 <connections/>
</ui>
